x-pack/plugins/ingest_pipelines/public/application/components/pipeline_processors_editor/components/pipeline_processors_editor_item/pipeline_processors_editor_item.tsx

```diff
@@ -94,7 +94,13 @@ export const PipelineProcessorsEditorItem: FunctionComponent<Props> = memo(
           disabled={isDisabled && !isMovingThisProcessor}
           label={label}
           aria-label={label}
-          onChange={() => (!isMovingThisProcessor ? onMove() : onCancelMove())}
+          onChange={() => {
+            if (!isMovingThisProcessor) {
```

Comment:
Nit: I find it easier to scan a condition if we're asserting against a positive statement, e.g. `if (isMovingThisProcessor)`. Seems like a nice readability win for a very small change.